src/components/CMessageRow.vue

```diff
@@ -69,7 +70,12 @@ export default {
     kwargs: Object,
     enqueuedDatetime: Date,
     startedDatetime: Date,
-    endDatetime: Date
+    endDatetime: Date,
+    currentDate: Date,
+    isExpandable: {
```

Comment:
this will help that we can disable the effects to expand a sub row when this is component is a child of group, this was done to meet the interface requirements in the mock up, as can be seen the complexity to display the styles incremented